modules/skybrowser/src/screenspaceskybrowser.cpp

```diff
@@ -88,6 +88,14 @@ namespace {
         openspace::properties::Property::Visibility::User
     };
 
+    // This `ScreenSpaceRenderable` is added by the Sky Browser feature of OpenSpace, to
```

Comment:
```suggestion
    // This `ScreenSpaceRenderable` is used to
```
?

Plus rejiggering the remainder of the text to be block-formatted